warp-contracts-plugin-signature/README.md

```diff
@@ -20,9 +38,8 @@ const warp = await WarpFactory.forMainnet().use(new EvmSignatureVerificationPlug
 
 ## Future plans
 
-1. Expand to NodeJs environment.
-2. Usage of other wallets (e.g. Solana) and plugins.
-3. Add option to deploy contracts using Warp Signature.
+1. Usage of other wallets (e.g. Solana) and plugins.
+2. Add option to deploy contracts using Warp Signature.
```

Comment:
point 2 can also be deleted as it's already done :)

Reply:
done!